BoardGameTrainer/Application/MainWindow.cs

```diff
@@ -22,12 +22,12 @@ internal class MainWindow : Gtk.Window
         private readonly BlockingCollection<Action> eventsQueue = new(new ConcurrentQueue<Action>());
 
         private CancellationTokenSource tokenSource = new();
-        private WindowState windowState = WindowState.Idle;
+        private WindowState windowState = WindowState.ProcessMovement;
```

Comment:
myślę, że jeśli używamy danego stanu w dwóch różnych sytuacjach, to albo musi mieć ogólniejszą nazwę (adekwatną do obu sytuacji), albo trzeba to rozwiązać jakoś inaczej